src/main/java/ch/uzh/ifi/hase/soprafs22/entity/User.java

```diff
@@ -115,10 +121,33 @@ public class User implements Serializable {
     public Gender getGender(){return this.gender; }
     public void setGender(Gender gender){this.gender = gender; }
 
-    public Game getActiveGame() {return activeGame;}
-    public void setActiveGame(Game activeGame) {this.activeGame = activeGame;}
+    public Game getActiveGame() {
+        if(!this.games.isEmpty() && this.games.get(games.size() - 1).getGameStatus()== GameStatus.ACTIVE){
+            return  this.games.get(games.size() - 1);
+        }
+            return null;
+    }
+
+    public void setActiveGame(Game activeGame) {
+        activeGame.setGameStatus(GameStatus.ACTIVE);
```

Comment:
Why do you (re-)set the game status here and not when creating the game?

Reply:
true! 
I was not sure, for what setActiveGame is needed, if it is needed. If it is for putting a past game active again, the status has to be set to active.
i do not now, if it makes more sence to just have the method ad game, isntead of ad active game

Reply:
added addGame on user